includes/class-wc-siftscience-admin.php

```diff
@@ -144,6 +144,14 @@ private function output_settings_debug() {
 		}
 
 		private function output_settings_reporting() {
+			if ( isset( $_GET[ 'reset_requested' ] ) ) {
```

Comment:
`reset_requested` isn't clear. Reset what? I think a better name would be `reset_guid` or something like that.

Reply:
ok